graphql/api.js

```diff
@@ -256,6 +257,38 @@ export const updateApplication = async (id, company, role, currentExcitement) =>
 
   const data = await res.json();
 
+  return {
+    data: getData(data),
+    errorMessage: getErrorMessage(null, data)
+  };
+};
+
+export const updateInterviewNotes = async (id, notes, date, type) => {
+  const query = `mutation updateInterviewNotes($id: ID!, $notes: String!, $date: String!, $type: String!) {
+    updateInterview(id: $id, data:{
+      date: $date
+      type: $type
+      notes: $notes
```

Comment:
does it need `date` and `type` to update?

Reply:
Yeah, for some reason GraphQL forces you to include a type's required props (in this case date and type) even if you're updating only one

Reply:
ok. interesting.